application/cmdbabstract.class.inc.php

```diff
@@ -2201,7 +2201,8 @@ public static function GetFormElementForField($oPage, $sClass, $sAttCode, $oAttD
 						{
 							$oPage->add_ready_script(
 								<<<EOF
-								$('#{$iId}').qtip( { content: $('#{$iId}').val(), show: 'mouseover', hide: 'mouseout', style: { name: 'dark', tip: 'bottomLeft' }, position: { corner: { target: 'topLeft', tooltip: 'bottomLeft' }, adjust: { y: -15}} } );
+								var escapedVal = $('<div/>').text($('#{$iId}').val()).html();
```

Comment:
```suggestion
								var sEscapedVal = $('<div/>').text($('#{$iId}').val()).html();
```
Our naming convention defines the variable type with a letter prefix, if you could add this little change!